LibC/arpa/inet.cpp

```diff
@@ -12,6 +14,7 @@ const char* inet_ntop(int af, const void* src, char* dst, socklen_t len)
         return nullptr;
     }
     auto* bytes = (const unsigned char*)src;
+    ASSERT(strlen((const char*)bytes) == INET_ADDRSTRLEN);
```

Comment:
This doesn't seem right, the longest IPv4 address string would be e.g "255.255.255.255" which is 15 characters.
Also, they might be shorter like "1.1.1.1" (7 characters).

Reply:
Oh wait, what is this code event.. isn't "bytes" a raw 32-bit IP address here?

Reply:
Sorry, that was completely braindead from reading the manpage and not really reading the manpage. Removed it :)